forage-android/src/main/java/com/joinforage/forage/android/core/Log.kt

```diff
@@ -53,6 +53,8 @@ internal interface Log {
                     .setBundleWithTraceEnabled(true)
                     .setLoggerName(LOGGER_NAME)
                     .build()
+
+                logger?.addAttribute("version", BuildConfig.VERSION)
```

Comment:
@devinmorgan @djoksimo Any thoughts on naming this something like "android_sdk_version" or something? Or are we cool with "version" being the naming convention.

Reply:
I'm cool with "version"